tests/installation/disable_grub_timeout.pm

```diff
@@ -50,6 +45,13 @@ sub run {
         assert_screen 'booting-section-selected';
         send_key 'ret';
     }
+
+    # Config bootloader is not be supported during an upgrade
+    if (get_var('UPGRADE') && (!is_sle('<15') || !is_leap('<15.0'))) {
```

Comment:
What's about using is_sle feature for version comparison instead of negation? E.g.
```perl
is_sle('>=15') || is_leap('>=15.0') || is_tumbleweed
```
I guess we will need is_tumbleweed, as current expression will return true in case UPGRADE is set and distri is TW. WDYT?

Reply:
Good idea, I have rebased the code.

Reply:
Hm, overall I prefer to avoid `is_tumbleweed` as that is not really a good version to check for. Rather we should accept the TW behaviour always as the default and handle exceptions differently, e.g. in this case handle the `is_sle('<15') || is_leap('<15')` case as exception and accept everything else as default.

If you agree, please change that. If you do not want to change this in this PR then please ping me or anyone else to merge directly as is.